test/python/qasm2/test_structure.py

```diff
@@ -39,10 +39,22 @@
 from . import gate_builder
 
 
-class TestEmpty(QiskitTestCase):
+@ddt.ddt
+class TestWhitespace(QiskitTestCase):
     def test_allows_empty(self):
         self.assertEqual(qiskit.qasm2.loads(""), QuantumCircuit())
 
+    @ddt.data("", "\n", "\r\n", "\n  ")
```

Comment:
Do you want to test `"\n\t"` and `"\r\n\t"` too? Just thinking of other whitespace combinations to make the testing here a bit more thorough.

Reply:
extra cases added in c410d75573256930f61c6e0ef489109c28fb3a68

Reply:
The only one of these cases that was actually buggy was the one with no whitespace at all, but it doesn't matter a great deal that there's additional cases.

Reply:
I assumed you were already afk, so I took the liberty of add them.

Reply:
That's ok, it's all _our_ code anyway.